src/VarUnit.cpp

```diff
@@ -179,19 +179,19 @@ QStringList VarUnit::shortVarName(TVar* var)
         return names;
     }
     names << var->getName();
-    TVar* p = var->getParent();
-    while (p && p->getName() != "_G") {
-        names.insert(0, p->getName());
-        p = p->getParent();
+    TVar* pParent = var->getParent();
+    while (pParent && pParent->getName() != "_G") {
```

Comment:
Oooh! `_G` looks as though it is a raw C string literal - so if `TVar::getName()` returns a `QString` type than this should be wrapped in a `qsl(`...`)` or a `QLatin1String(`...`)` wrapper...! 🤓 

Just an observation,  no action needed for this PR.

Reply:
Applied.